README.md

```diff
@@ -159,8 +159,9 @@ pub async fn schema(ctx) {
 ### Prepared statements
 
 Calling `ctx.execute` is not optimal, because it doesn't use prepared statements. You can prepare statements and
-register them on the context object in the `prepare`
-function:
+register them on the context object in the `prepare` function. 
+They will be executed during [the load step](#populating-the-database) before the actual database population.
```

Comment:
Prepare is called before both load and run as well. Can you update the comment to reflect that?